app/nasa_collaboration/page.js

```diff
@@ -18,14 +17,23 @@ export const NasaCollaboration = () => {
 
   useEffect(() => {
     const fetchRoverPhotos = async () => {
-      const roverPhotoResponse = await fetch(NASA_URLs.marsRoverPhoto).then(response => response.json());
+      const roverPhotoResponse = await fetch(NASA_URLs.marsRoverPhoto).then(
+        (response) => response.json()
+      );
       setRoverPhoto(roverPhotoResponse);
     };
 
     fetchRoverPhotos();
+  }, []);
 
-    // TASK - React 1 week 3 
-    // fetch the extra data for NASA_URLs.astronomyPicOfTheDay and save it to the dailyImg state variable
+  useEffect(() => {
+    const fetchDailyImages = async () => {
```

Comment:
```suggestion
    const fetchDailyImage = async () => {
```
Just a small note. 😄 It is only one image after all.